webknossos-datastore/app/com/scalableminds/webknossos/datastore/datavault/VaultPath.scala

```diff
@@ -0,0 +1,76 @@
+package com.scalableminds.webknossos.datastore.datavault
+
+import com.scalableminds.util.io.ZipIO
+import com.scalableminds.webknossos.datastore.storage.FileSystemCredential
+import net.liftweb.util.Helpers.tryo
+
+import java.net.URI
+import java.nio.file.{FileSystem, LinkOption, Path, Paths, WatchEvent, WatchKey, WatchService}
+
+/*
+VaultPath implements Path so that a drop in replacement is possible while continuing to use Paths for local storage.
+This class does not implement all relevant methods and it might be a good idea to remove the inheritance on Path in the
+future.
+ */
+
+class VaultPath(uri: URI, dataVault: DataVault, fileSystemCredentialOpt: Option[FileSystemCredential]) extends Path {
+
+  def get(key: String, range: Option[Range] = None): Array[Byte] = dataVault.get(key, this, range)
+
+  def tryGet(key: String, range: Option[Range] = None): Option[Array[Byte]] =
```

Comment:
Is there any use case where we want to use get and not tryGet? If not, let’s just make tryGet the default and the other private.

Long-term, my guess is that we want to use Foxes for all of the error handling, but that does not have to happen now. I wrote follow-up issue https://github.com/scalableminds/webknossos/issues/6906